jenkins/catapult/environment.py

```diff
@@ -28,6 +28,18 @@ def system_bin_path(self):
 
 		raise RuntimeError('unable to detect system bin path')
 
+	@property
+	def local_lib_path(self):
+		if self.dry_run:
+			return '<LOCAL_LIB_PATH>'
+
+		for descriptor in [('fedora', '/usr/local/lib64'), ('ubuntu', '/usr/local/lib')]:
```

Comment:
i guess we don't call this for debian ?

Reply:
yea, it does get call for Debian, but it will match ubuntu since it checks for path existing.  Not sure if there is a better way.

Reply:
ok, that's probably fine